Add Site-to-Site Connection Option to Connect Astro to GCP Data Sources (#1963)
astronomer/docs: astro/connect-gcp.md

</TabItem>

<TabItem value="Site">

> A site-to-site virtual private network (VPN) connects two or more networks to each other and provides a stable and secure connection between a GCP VPN and Astro. To implement this connection type, Astronomer supports uses the [Google Cloud VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/overview).

Reviewer: What about:
"A site-to-site virtual private network (VPN) connects two networks on secure manner. To implement this connection type between Astro and  other GCP network, Astronomer supports uses the [Google Cloud VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/overview)."

> A site-to-site virtual private network (VPN) connects two or more networks to each other and provides a stable and secure connection between a GCP VPN and Astro. To implement this connection type, Astronomer supports uses the [Google Cloud VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/overview).

Reviewer 2: ```suggestion
A site-to-site virtual private network (VPN) connects two networks to each other in a secure manner. To implement this connection type, Astronomer uses the [Google Cloud VPN](https://cloud.google.com/network-connectivity/docs/vpn/concepts/overview).
```
